tests/test_io/test_mongodb.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+
+"""Tests for the jsonschema node validation."""
+
+import copy
+from typing import Any, List, Mapping, Tuple
+import unittest
+
+import pymongo
+from pymongo.collection import Collection
+
+from pybel import BELGraph, to_nodelink
+from pybel.dsl import Entity, ComplexAbundance, Gene, Protein, Rna
+from pybel.io.mongodb import (
+    _entity_to_dict, 
+    to_mongodb,
+    find_nodes,
+    get_edges_from_node,
+    get_edges_from_criteria
+)
+from pybel.testing.utils import n
+
+g1 = Gene('hgnc', '1')
+r1 = Rna('hgnc', '1')
+p1 = Protein('hgnc', '1')
+g2 = Gene('hgnc', '2')
+r2 = Rna('hgnc', '2')
+p2 = Protein('hgnc', '2')
+g3 = Gene('hgnc', '3')
+p3 = Protein('hgnc', '3')
+
+ca = ComplexAbundance([p1, g2], name='ca', namespace='hgnc')
+
+client = pymongo.MongoClient()
+TEST_DB = client['test_db']
+TEST_COLLECTION = TEST_DB['test_collection']
+TEST_COLLECTION.drop()
+
+
+class TestMongoDB(unittest.TestCase):
+    """Tests for the MongoDB exporting and querying."""
+    def setUp(self):
+        """Create and export a test graph."""
```

Comment:
create an instance of the mongo client here, and if it fails, use `self.skip()` - this will make it pass on appveyor, where mongodb is not available